src/actions/guides/getting-started/installing.cr

```diff
@@ -6,22 +6,36 @@ class Guides::GettingStarted::Installing < GuideAction
     "Installing Lucky"
   end
 
-  def markdown
+  def markdown : String
     <<-MD
     #{permalink(ANCHOR_INSTALL_REQUIRED_DEPENDENCIES)}
     ## Install Required Dependencies
 
-    To get Lucky, you need to install these first.
+    ### Crystal v0.31.1
```

Comment:
Gave each dep a subheading to make it more scannable. Would love thoughts on this

Reply:
Yeah love it. Good headings help guide you through the doc. These are good.